src/Persistence/Templates/WellKnownTemplateIds.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.PowerPlatform.PowerApps.Persistence.Templates;
+
+public static class WellKnownTemplateIds
+{
+    public const string appinfo = "http://microsoft.com/appmagic/appinfo";
```

Comment:
also, why the casing is not consisten?

Reply:
the casing is consistent with the case in the template uri. I know this isn't normal for .net naming. It's a reminder to the consumer what it's value is.

But I can make them all start with upper case, and actually, I can make them CamelCase too.